recipes/python-vagrant/meta.yaml

```
@@ -0,0 +1,38 @@
+{% set name = "python-vagrant" %}
+{% set version = "0.5.15" %}
+
+ package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+ source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: af9a8a9802d382d45dbea96aa3cfbe77c6e6ad65b3fe7b7c799d41ab988179c6
+
+ build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```

`--no-deps --ignore-installed` are included by default
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv "
```